dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/calculate-fixed-fee.test.ts

```diff
@@ -0,0 +1,42 @@
+import { addDays } from 'date-fns';
+import { calculateFixedFee, getNumberOfDaysInCoverPeriod } from './calculate-fixed-fee';
+import { calculateFixedFeeFromDaysRemaining } from '../../../../../helpers/calculate-fixed-fee-from-days-remaining';
+
+describe('helpers/calculate-fixed-fee', () => {
+  describe('getNumberOfDaysInCoverPeriod', () => {
+    it('should return the difference in days between 2 dates', () => {
+      const difference = 12;
+
+      const today = new Date();
+      const future = addDays(today, difference);
+
+      const result = getNumberOfDaysInCoverPeriod(today, future);
+
+      expect(result).toEqual(difference);
+    });
+  });
+
+  describe('calculateFixedFee', () => {
```

Comment:
Do we still have the tests that test the actual values of the calculateFixedFee with the raw numbers?

Reply:
yes we do in dtfs-central-api\src\services\state-machines\fee-record\event-handlers\helpers\calculate-fixed-fee.test.ts and the new test I have added